### PR TITLE
Add libgda 5.2.4

### PR DESCRIPTION
**Pull request by dsendkowski**

### CODE REVIEW

File: components/library/libgda/libgda.p5m [hunk 1]
```diff
@@ -0,0 +1,658 @@
...
+link path=usr/bin/gda-list-config target=gda-list-config-5.0
+file path=usr/bin/gda-list-config-5.0
+link path=usr/bin/gda-list-server-op target=gda-list-server-op-5.0
+file path=usr/bin/gda-list-server-op-5.0
+link path=usr/bin/gda-sql target=gda-sql-5.0
+file path=usr/bin/gda-sql-5.0
+file path=usr/bin/gda-test-connection-5.0
+file path=usr/bin/gdaui-demo-5.0
+file path=usr/etc/libgda-5.0/config
+file path=usr/etc/libgda-5.0/sales_test.db
```
Comment (pyhalov) [inline components/library/libgda/libgda.p5m]: Specify sysconfigdir usr/etc...

Comment (dsendkowski) [reply components/library/libgda/libgda.p5m]: OK, done it but not sure if that is what you meant.

File: components/library/libgda/libgda.p5m [hunk 2]
```diff
@@ -0,0 +1,660 @@
...
+link path=usr/bin/gda-list-server-op target=gda-list-server-op-5.0
+file path=usr/bin/gda-list-server-op-5.0
+link path=usr/bin/gda-sql target=gda-sql-5.0
+file path=usr/bin/gda-sql-5.0
+file path=usr/bin/gda-test-connection-5.0
+file path=usr/bin/gdaui-demo-5.0
+file path=etc/$(MACH64)/libgda-5.0/config
+file path=etc/$(MACH64)/libgda-5.0/sales_test.db
+file path=etc/libgda-5.0/config
+file path=etc/libgda-5.0/sales_test.db
```
Comment (pyhalov) [inline components/library/libgda/libgda.p5m]: Are these actual config and db or just examples? If just examples, perhaps, move them out of /etc.

Comment (dsendkowski) [reply components/library/libgda/libgda.p5m]: Yeah, it seems these files are some examples. Where should I move them?

Comment (pyhalov) [reply components/library/libgda/libgda.p5m]: Perhaps, /usr/share/libgda/examples or /usr/share/libgda-5.0/examples ?

Comment (dsendkowski) [reply components/library/libgda/libgda.p5m]: Ok, I moved them to `/usr/share/libgda-5.0/examples` directory.

**Comment by pyhalov** [comment]
```
Couldn't find 'usr/share/glade/catalogs/gdaui-catalog.xml' in any of the specified search directories:
...
Couldn't find 'usr/share/glade/pixmaps/widget-gdaui-gdauibasicform.png' in any of the specified search directories:
...
Couldn't find 'usr/share/glade/pixmaps/widget-gdaui-gdauicombo.png' in any of the specified search directories:
 ...
```
Also, examples are the same for amd64/x86 , so no need to deliver 2 copies.

**Comment by dsendkowski** [comment]
When do you get these errors? These files are provided by the component.

**Comment by pyhalov** [comment]
When build it on latest OI

**Comment by dsendkowski** [comment]
So, just to clarify the build fails and stops, right? Could you paste a little more, including the last log entries?
I also build this on the latest OI and do not get these errors. I cannot reproduce it.

**Comment by pyhalov** [comment]
http://buildzone.oi-build.r61.net/libgda.publish.log

**Comment by xen0l** [comment]
@dsendkowski any news on this?

**Comment by dsendkowski** [comment]
I couldn't replicate these errors. Maybe we should either postpone (until I have time to investigate it more thoroughly) or close it.

**Comment by pyhalov** [comment]
Do you happen to have gladeui-2.0.pc installed ?

**Comment by pyhalov** [comment]
@dsendkowski ping

**Comment by dsendkowski** [comment]
I'm really sorry but I'm not able to continue my work on this particular subject right now due to lack of time. Please, feel free to close this PR since it is rather dead now.

**Comment by xen0l** [comment]
@dsendkowski if you find more time to pursue this, then feel free to reopen.

**Pull request closed without merging**